packages/flutter_dynamic_forms_components/lib/src/components/slider/reactive_slider_renderer.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_dynamic_forms/flutter_dynamic_forms.dart';
+
+import 'slider.g.dart' as model;
+
+class ReactiveSliderRenderer extends FormElementRenderer<model.Slider> {
+  @override
+  Widget render(
+      model.Slider element,
+      BuildContext context,
+      FormElementEventDispatcherFunction dispatcher,
+      FormElementRendererFunction renderer) {
+    return StreamBuilder(
+      stream: element.valueChanged,
```

Comment:
`element.propertyChanged` is better because it will rebuild if any property will be changed on the model, not only the value.